swagger.yaml

```diff
@@ -0,0 +1,633 @@
+swagger: '2.0'
```

Comment:
Do we want to use OAS 3 instead (see [here](https://swagger.io/docs/specification/basic-structure/))? cc @lvrach   

```suggestion
openapi: 3.0.3
```

Reply:
yes lets use openAPI

Reply:
@pras-stack I see that you reposted this for review (Slack) but this file hasn't been updated to use the OpenAPI 3 specs, right?

Reply:
yes done.